### PR TITLE
fixed joined users mentioned

### PR DESCRIPTION
**Pull request by abughalib**
fixed team listing when users leave/join the Team.

### CODE REVIEW

File: handlers/interactions/buttonInteractions.ts [hunk 1]
```diff
@@ -400,7 +403,10 @@ async function interactionButtonHandler(interaction: ButtonInteraction) {
 function removeIndex(arri: Array<string>, elem: string) {
   let array: Array<string> = [];
 
-  for (let i = 0; i < arri.length && arri[i] != elem; i += 1) {
+  for (let i = 0; i < arri.length; i += 1) {
```
Comment (AshAman999) [inline handlers/interactions/buttonInteractions.ts:408]:
```suggestion
  for (let i = 0; i < arri.length; i += 1) {
  if(arri[i] !== elem){
  array.push(arri[i];
```

File: handlers/interactions/buttonInteractions.ts [hunk 2]
```diff
@@ -400,7 +403,10 @@ async function interactionButtonHandler(interaction: ButtonInteraction) {
 function removeIndex(arri: Array<string>, elem: string) {
```
Comment (AshAman999) [inline handlers/interactions/buttonInteractions.ts]: elem ? 
What does it even mean, could you make it more readable?

**Review by AshAman999 — CHANGES_REQUESTED** [review]

File: handlers/interactions/buttonInteractions.ts (hunk 1 above)
Comment (abughalib) [reply handlers/interactions/buttonInteractions.ts:408]: They are both same almost the same login, I would accept your suggestion

File: handlers/interactions/buttonInteractions.ts (hunk 2 above)
Comment (abughalib) [reply handlers/interactions/buttonInteractions.ts]: users

**Review by AshAman999 — APPROVED** [review]

**Pull request merged**